src/resources/2016/Witchess.ts

```diff
@@ -51,5 +52,6 @@ export function fightPiece(piece: Monster): string {
   ) {
     throw new Error("Failed to start fight.");
   }
+  if (macro) return runCombat(macro.toString());
   return runCombat();
```

Comment:
```suggestion
  return runCombat(macro?.toString());
```
Does this look better?

Reply:
Yes! I didn't know you could do that.